### PR TITLE
feat: added hairStyle "beanie"

### PR DESCRIPTION
**Pull request by itskaustubh**
Added a beanie for those who don't identify with any of the hairstyles. 
Suggestion/feedback regarding the design would be appreciated! :)

beanie_preview

### CODE REVIEW

**Comment by wjq990112** [comment]
LGTM

File: src/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const sex = ["man", "woman"];
 const faceColor = ["#F9C9B6", "#AC6651"];
 const earSize = ["small", "big"];
 const hairColor = ["#000", "#fff", "#77311D", "#FC909F", "#D2EFF3", "#506AF4", "#F48150"];
-const hairStyleMan = ["normal", "thick", "mohawk"];
+const hairStyleMan = ["normal", "thick", "mohawk","beanie"];
```
Comment (wjq990112) [inline src/index.js:18]: Lost space between `,` and `"`.

File: src/index.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default class ReactNiceAvatar extends Component {
     faceColor: PropTypes.string,
     earSize: PropTypes.oneOf(["small", "big"]),
     hairColor: PropTypes.string,
-    hairStyle: PropTypes.oneOf(["normal", "thick", "mohawk", "womanLong", "womanShort"]),
+    hairStyle: PropTypes.oneOf(["normal", "thick", "mohawk", "womanLong", "womanShort","beanie"]),
```
Comment (wjq990112) [inline src/index.js:49]: Same as above.

Comment (itskaustubh) [reply src/index.js:49]: Thanks for pointing it out :)

**Comment by wwayne** [comment]
Thanks : )

**Pull request merged**